move-mutator/src/operators/binary.rs

```diff
@@ -85,8 +89,33 @@ impl MutationOperator for Binary {
             },
         };
 
+        let is_left_exp_zero = contains_value_zero(self.exps[0].exp.as_ref());
+        let is_right_exp_zero = contains_value_zero(self.exps[1].exp.as_ref());
+
         ops.into_iter()
             .filter(|v| cur_op != *v)
+            .filter(|v| match cur_op {
+                // All below mutants would lead to the same code logic and would become
+                // false-positive results.
+
+                // x == 0 should never mutate to x <= 0
+                "==" if is_right_exp_zero && *v == "<=" => false,
+                // 0 == x should never mutate to 0 => x
```

Comment:
To be exact with the operator below:

```suggestion
                // 0 == x should never mutate to 0 >= x
```

Reply:
Done